gradle/test.gradle

```
@@ -26,8 +26,9 @@ test {
     }
 }
 
-tasks.register("integrationTesting", Test) {
+tasks.register("integrationTestingPostgresql", Test) {
```

Please add an alias for backwards compatibility with the pipeline and ease of use.
```suggestion
tasks.register("integrationTesting") {
    dependsOn("integrationTestingPostgresql")
}

tasks.register("integrationTestingPostgresql", Test) {
```